test-guru/app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,61 @@
+class QuestionsController < ApplicationController
+  before_action :find_test, only: [:index, :create, :new]
+  before_action :find_question, only: [:show, :destroy, :edit, :update]
+
+  rescue_from ActiveRecord::RecordNotFound, with: :rescue_with_question_not_found
+
+
+  def index
+    @questions = @test.questions
+  end
+
+  def show
+    @test = Test.find(@question.test_id)
+  end
+
+  def create
+    @question = @test.questions.new(question_params)
+    if @question.save then
```

Comment:
```suggestion
    if @question.save
```
`then` пришут разве что в однострочных конструкциях 
```ruby
if .... then ....
elsiv .... then ....
```